assets/src/edit-story/components/library/panes/pageLayouts/pageLayout.js

```diff
@@ -142,8 +142,8 @@ function PageLayout(props) {
           onClick={handleClick}
           isActive={isActive}
           aria-label={page.title}
-          title={page.title}
```

Comment:
This was actually added specifically after the bug bash because without this when you hover, it passes down an ascendant title of 'Page layouts' which is weird and confusing to see when you're hovering. If you can find another way around that that's cool, but this should come back to fix that weirdness.

Reply:
how _odd_, lemme look again. I was under the impression we don't want the gray title that shows on default with a title attribute.

Reply:
Restored the title for the time being

Reply:
Is that because `PageLayoutsParentContainer` has a `title` prop? 

https://github.com/google/web-stories-wp/blob/665c14fa8bd6355bf322aed7dfcf814549fe4803/assets/src/edit-story/components/library/panes/pageLayouts/pageLayoutsPane.js#L128-L131

Doesn't really make sense to me as it serves no purpose.

Reply:
With that, I'd remove the `title={page.title}` here as well.

Reply:
yup, thank you pascal - that fixed it.